### ISSUE TITLE
Maps

### ISSUE THREAD

**Issue by RonaldDijkstra** [DefactoSoftware/website#1630]
Earlier we had our location on the about us page with Google Maps. This was removed due to troubles with the implementation (we had to pay for it or something)

We can use something else: 

https://www.openstreetmap.org/#map=8/52.154/5.295 with https://leafletjs.com/ 

Which is actually what is used in detroit.

**Comment by lennartdefacto** [comment]
True, if I remember correctly Google will now bill you for the Maps JavaScript API.
